src/ast/mod.rs

```diff
@@ -750,6 +750,20 @@ pub enum Statement {
         /// CSV HEADER
         csv_header: bool,
     },
+    CopyTo {
```

Comment:
It looks to me like both `CopyFrom` and `CopyTo` share all the same fields.

What would you think about just adding a new field to `Copy` rather than making `CopyTo` and `CopyFrom`?

Something like:


```rust
  Copy {
        /// TABLE
        table_name: ObjectName,

...
        /// CSV HEADER
        csv_header: bool,
        /// If true, represents COPY TO, else COPY FROM
        to: bool,        
    },
```

It looks to me like the only difference between `COPY TO` and `COPY FROM` is that COPY FROM also supports a `WHERE` clause in postgres.

Reply:
Indeed, they are effectively the same asides from COPY TO taking a query as well. and im definitely fine with updating to your proposal.  I was naively under the impression that each statement should have its own variant.

btw below is from postgres.
```
COPY table_name [ ( column_name [, ...] ) ]
    FROM { 'filename' | STDIN }
    [ [ WITH ] ( option [, ...] ) ]

COPY { table_name [ ( column_name [, ...] ) ] | ( query ) }
    TO { 'filename' | STDOUT }
    [ [ WITH ] ( option [, ...] ) ]

where option can be one of:

    FORMAT format_name
    OIDS [ boolean ]
    DELIMITER 'delimiter_character'
    NULL 'null_string'
    HEADER [ boolean ]
    QUOTE 'quote_character'
    ESCAPE 'escape_character'
    FORCE_QUOTE { ( column_name [, ...] ) | * }
    FORCE_NOT_NULL ( column_name [, ...] )
    ENCODING 'encoding_name'
```

Reply:
> Indeed, they are effectively the same asides from COPY TO taking a query as well. and im definitely fine with updating to your proposal. I was naively under the impression that each statement should have its own variant.

I guess I was thinking the `COPY` was the statement rather than `COPY FROM` and `COPY TO` being separate statements